### PR TITLE
feat(storage.cold-archive): update cold-archive tracking

### PR DESCRIPTION
**Pull request by brofik**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/MANAGER-9069` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9194 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by stif59100 — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 1]
```diff
@@ -65,7 +65,7 @@
                 <!--Action: add user-->
                 <oui-action-menu-item
                     data-ng-if="$ctrl.constructor.isActionAddUserAvailable($row)"
-                    data-on-click="$ctrl.goToAddUserToContainer($row)"
+                    data-on-click="$ctrl.onClickOnAddUsertoContainer($row)"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:68]:
```suggestion
                    data-on-click="$ctrl.onAddUserToContainerClick($row)"
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 2]
```diff
@@ -75,7 +75,7 @@
                 <!--Action: archive container-->
                 <oui-action-menu-item
                     data-ng-if="$ctrl.constructor.isActionArchiveAvailable($row)"
-                    data-on-click="$ctrl.goToArchiveContainer($row)"
+                    data-on-click="$ctrl.onClickOnArchiveContainer($row)"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:78]:
```suggestion
                    data-on-click="$ctrl.onArchiveContainerClick($row)"
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 3]
```diff
@@ -85,7 +85,7 @@
                 <!--Action: restored container-->
                 <oui-action-menu-item
                     data-ng-if="$ctrl.constructor.isActionRestoredAvailable($row)"
-                    data-on-click="$ctrl.goToRestoreContainer($row)"
+                    data-on-click="$ctrl.onClickOnRestoreContainer($row)"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:88]:
```suggestion
                    data-on-click="$ctrl.onRestoreContainerClick($row)"
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 4]
```diff
@@ -116,10 +116,7 @@
         <oui-datagrid-topbar>
             <button
                 class="oui-button oui-button_secondary"
-                data-ng-click="$ctrl.goToAddColdArchive()"
-                data-track-on="click"
-                data-track-name="PCI_PROJECTS_CONTAINERS_ADD"
-                data-track-type="navigation"
+                data-ng-click="$ctrl.onClickOnAddContainer()"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:119]:
```suggestion
                data-ng-click="$ctrl.onAddContainerClick()"
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js [hunk 5]
```diff
@@ -39,8 +40,50 @@ export default class PciStoragesColdArchiveContainersController {
...
+  }
+
+  onFlushContainerClick(container) {
+    this.trackContainerslClick(
+      COLD_ARCHIVE_TRACKING.CONTAINERS.FLUSH_CONTAINER,
+    );
+    return this.goToFlushArchive(container);
+  }
+
+  onAddUsertoContainerClick(container) {
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.controller.js:79]:
```suggestion
  onAddUserToContainerClick(container) {
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html [hunk 6]
```diff
@@ -65,7 +65,7 @@
                 <!--Action: add user-->
                 <oui-action-menu-item
                     data-ng-if="$ctrl.constructor.isActionAddUserAvailable($row)"
-                    data-on-click="$ctrl.goToAddUserToContainer($row)"
+                    data-on-click="$ctrl.onAddUsertoContainerClick($row)"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html:68]:
```suggestion
                    data-on-click="$ctrl.onAddUserToContainerClick($row)"
```

File: packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/import/import.controller.js [hunk 7]
```diff
@@ -75,10 +73,17 @@ export default class PciBlockStorageContainersContainerObjectAddController {
...
-    this.atInternet.trackClick({
-      name: `${this.trackingPrefix}${TRACKING.IMPORT_POLICY}::cancel`,
-      type: 'action',
-    });
+    this.trackImportModalClick(COLD_ARCHIVE_TRACKING.ACTIONS.CANCEL);
     return this.goBack();
   }
+
+  trackImportModalPage(action) {
+    const hit = `${COLD_ARCHIVE_TRACKING.USER.MAIN}::${COLD_ARCHIVE_TRACKING.USER.ACTIONS.IMPORT_POLICY}_${action}`;
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/import/import.controller.js:81]: for readability, may be better to have this:
```suggestion
    const {MAIN, ACTIONS} =  COLD_ARCHIVE_TRACKING.USER;
    const hit = `${MAIN}::${ACTIONS.IMPORT_POLICY}_${action}`;
```

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**